source/content/guides/edge-integrations/02-configuration-overview.md

```diff
@@ -28,9 +30,12 @@ When you personalize with AGCDN, the content variant generated by the set of con
 
 Your CMS processes the original content, makes a determination, and displays a first-time rendering of the content. Once the content is rendered for the first time, it’s being saved in the CDN cache layer alongside the conditional header; your CMS isn’t involved until the cache expires, content is updated, or the conditional variant doesn’t exist in the cache layer.
 
-## Configuration
+## Cache Overview
 
+A `Vary` response header tells caches that a particular header (or headers) from the request should be used to construct a secondary cache key that must match in order to use the cached response to answer subsequent requests.
 
+For example, when the Vary header is set for Interest, the a server produces versions of the same webpage with different Interests and serves them to visitor's browsers based on their locally defined interests.
```

Comment:
Hyphenate "locally-defined" to show possession.

```suggestion
For example, when the Vary header is set for Interest, the a server produces versions of the same webpage with different Interests and serves them to visitor's browsers based on their locally-defined interests.
```

Reply:
Addressed in [8b7ae81](https://github.com/pantheon-systems/documentation/pull/7186/commits/8b7ae81a4e4d560c9af7cf5e14e072036385d502).